### PR TITLE
Feature/door sensor

### PR DESCRIPTION
**Pull request by maximekabel**
implemented door sensor

### CODE REVIEW

File: cosmos/views.py [hunk 1]
```diff
@@ -401,3 +404,18 @@ def news_list(request):
...
+
+def update_door_status(request):
+    print(request.GET)
+    try:
+        token = request.GET.get("access_token")
+        status = request.GET.get("status")
+
+        if Token.objects.filter(token=token).exists():
+            os.environ["DOOR_STATUS"] = str(status)
+            print(os.environ.get("DOOR_STATUS", "0"))
```
Comment (dasyad00) [inline cosmos/views.py:417]: can the print statements be removed prior to merging?

**Review by dasyad00 — APPROVED** [review]

File: cosmos/models.py [hunk 2]
```diff
@@ -117,3 +117,11 @@ class Partner(models.Model):
 
     def __str__(self):
         return "Partner: {" + self.name + "}"
+
+
+class Token(models.Model):
+    token = models.CharField(max_length=100)
+    device = models.CharField(max_length=50)  # the target entity that uses this token
```
Comment (bcvandendool) [inline cosmos/models.py:124]: Unnecessary, only one device needed

File: cosmos/views.py [hunk 3]
```diff
@@ -32,6 +33,7 @@ def index(request):
     nationalities = Profile.objects.values("nationality").distinct().count()
     active_years = int((datetime.date.today() - FOUNDING_DATE).days // 365.25)
     events_amount = "20+"
+    door_status = int(os.environ.get("DOOR_STATUS", "0"))
```
Comment (bcvandendool) [inline cosmos/views.py:36]: pls no os environment stuff

**Review by bcvandendool — CHANGES_REQUESTED** [review]

**Review by bcvandendool — CHANGES_REQUESTED** [review]
Write tests

**Pull request merged**